models/validators/RequiredValidator.cfc

```diff
@@ -69,7 +69,11 @@ component
 	/**
 	 * Verify if the target value has value
 	 */
-	boolean function hasValue( required targetValue ){
+	boolean function hasValue( any targetValue ){
```

Comment:
I actually think we can remove this function because we have the new function `isNullOrEmpty()` in the `BaseValidator` object:

```
/**
	 * check if field is NOT null and has length
	 *
	 * @targetValue the value to check for nullness/emptyness
	 * @return boolean
	 */
	boolean function isNullOrEmpty( required any targetValue ){
		return isNull( arguments.targetValue ) || (
			isSimpleValue( arguments.targetValue ) && !len( arguments.targetValue )
		);
	}
```

Thoughts?

Reply:
`hasValue` handles more use cases, like Queries and Objects.  I'm not sure if they are needed, but they aren't quite the same functionality.

Reply:
Ahh I see.  Should we incorporate this into the base object?